rcjaRegistration/events/models.py

```diff
@@ -384,6 +385,9 @@ def get_absolute_url(self):
     def boolWorkshop(self):
         return self.eventType == 'workshop'
 
+    def bleachedEventDetails(self):
+        return mark_safe(bleach.clean(self.eventDetails))
```

Comment:
Just noting here that bleach.clean without any attributes will default to:

Allowed tags: bleach.ALLOWED_TAGS (a,abbr,acronym,b,blockquote,code,em,i,li,ol,strong,ul)
Allowed styles: []
Allowed Attributes: bleach.ALLOWED_ATTRIBUTES (href, title)

I believe this will also allow any link through, which should be fine? Unelss we want to limit it to a specific set of domains, which we can also do.

Reply:
Note: img is not an allowed tag, so users will not be able to embed images in their descriptions. (probably desired behaviour)

Reply:
Yes this is my understanding of the behaviour. Links are allowed which is probably desired but embedding images probably not. Links allows for files to be linked to (Google Drive etc), although will probably add a built in file upload system at some point (see #473). Can also upload files to the website and link to them.